composer.json

```diff
@@ -37,7 +38,8 @@
         }
     },
     "scripts": {
-        "tests": "vendor/bin/phpunit -c phpunit.xml.dist"
+        "tests": "vendor/bin/phpunit -c phpunit.xml.dist",
+        "fix": "vendor/bin/php-cs-fixer fix"
```

Comment:
I'd propose to have separate script entry for checking and fixing. And also a separate entry to execute all qa-related scripts. The check-script should also display the diff to help finding the reported errors. The workflow action should be updated to use the qa-script.

Something like this...
```php
    "qa": "composer check-cs && composer stan && composer test",
    "check-cs": "./vendor/bin/php-cs-fixer fix --dry-run --diff",
    "fix-cs": "./vendor/bin/php-cs-fixer fix",
    "test": "./vendor/bin/phpunit tests",
    "stan": "./vendor/bin/phpstan analyse",
```